core/tee/tee_svc_cryp.c

```diff
@@ -3785,27 +3942,29 @@ TEE_Result syscall_cryp_derive_key(unsigned long state,
 		res = TEE_ERROR_NOT_SUPPORTED;
 
 out:
+	for (i = param_count; i > 0; i--)
+		bb_free(bbufs[i].buf, bbufs[i].len);
```

Comment:
There is no need to call `bb_free()` on these buffers, the next thing to happen is that the bounce buffers are reset anyway.

Reply:
Updated